src/openrct2/peep/Peep.cpp

```diff
@@ -989,13 +989,20 @@ static void sub_68F41A(rct_peep * peep, sint32 index)
 
         if (peep->peep_flags & PEEP_FLAGS_EXPLODE && peep->x != LOCATION_NULL)
         {
-            audio_play_sound_at_location(SOUND_CRASH, peep->x, peep->y, peep->z);
+            if (peep->state == PEEP_STATE_WALKING || peep->state == PEEP_STATE_SITTING)
```

Comment:
You might be better off just leaving this as `if (peep->state == PEEP_STATE_WALKING)`. While the sitting state probably doesn't cause issues, there's not that much of a reason to include it either. The spectacle works just fine with only the walking peeps.

Reply:
I (and some other players i know) typically use this cheat to keep the numbers of guests a bit lower when a park is overcrowded, so the more peeps that are affected, the better 😁